src/v2/ion/ui-schema/ion-string-handler.js

```diff
@@ -53,15 +53,17 @@ const timezoneUISchema = {
 
 const shouldRenderAsRadio = (name) => name.indexOf('methodType') >= 0 || name.indexOf('channel') >= 0;
 
+const optionalType = ['text', 'radio', 'select'];
+
 const populateUISchemaForDisplay = (uiSchema, ionField) => {
-  let display = ionField.value.value;
+  const display = ionField?.value?.value;
   uiSchema.type = display.inputType;
   if (display.inputType === 'radio') {
     uiSchema.options = display.options;
   } else if (display.inputType === 'select') {
     uiSchema.wide = true;
     //it will create a placeholder for dropdowns, by default it will show 'Select an Option'
-    uiSchema.options = Object.assign({0: ''}, ionOptionsToUiOptions(display.options));
+    uiSchema.options = Object.assign({'': ''}, ionOptionsToUiOptions(display.options));
```

Comment:
Is this the change that made inline errors shows up ?

Reply:
Yes, it is only for the dropdown. The radio button collection was working fine.